kubeflow/gcp/setup_backend.sh

```diff
@@ -22,9 +22,18 @@ gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS
 gcloud config list
 
 NODE_PORT=$(kubectl --namespace=${NAMESPACE} get svc ${SERVICE} -o jsonpath='{.spec.ports[0].nodePort}')
+echo "node port is ${NODE_PORT}"
+
+while [[ -z ${BACKENDS} ]]; do
+  BACKENDS=$(kubectl --namespace=${NAMESPACE} get ingress ${SERVICE}-ingress -o jsonpath='{.metadata.annotations.ingress\.kubernetes\.io/backends}')
+  echo "fetching backends info: ${BACKENDS}"
+  sleep 2
+done
+BACKEND_NAME=$(echo $BACKENDS | grep -o "k8s-be-${NODE_PORT}--[0-9a-z]\+")
+echo "backend name is ${BACKEND_NAME}"
```

Comment:
should we check backend_name is not empty ?

Reply:
done.